netlify/edge-functions/mcp-server.ts

```diff
@@ -95,7 +95,13 @@ function formatResponse(data: unknown): {
 }
 
 // Netlify Edge Function Handler
-export default async function handler(req: Request) {
+export default async function handler(req: Request, { url }: Context) {
+  if (req.method !== "POST" && url.pathname === "/mcp") {
+    return Response.redirect(
+      "https://docs.astro.build/en/guides/build-with-ai/",
```

Comment:
We could also redirect to "/" instead of other docs:

```suggestion
      "https://mcp.docs.astro.build",
```

Reply:
I think docs makes sense. That's all the main page links to anyway